pixmappy/PixelMapCollection.py

```diff
@@ -188,23 +209,22 @@ def __init__(self,name,**kwargs):
             else:
                 # ordering is 1., y, y^2, y^3 ... , x, xy, xy^2,
                 c = v.reshape(xOrder, yOrder)
-            self.coeffs.append(np.array(c))
-        return
-    def __call__(self, xy, c=None):
-        xy_ = np.array(xy)
-        is1d = xy_.ndim==1
-        xy_ = np.atleast_2d(xy_)
-        xyscaled = (xy_ - self.shift)*self.scale
-        xyw = np.zeros_like(xy_)
-        xyw[:,0] = np.polynomial.polynomial.polyval2d(xyscaled[:,0],xyscaled[:,1],
-                                                     self.coeffs[0])
-        xyw[:,1] = np.polynomial.polynomial.polyval2d(xyscaled[:,0],xyscaled[:,1],
-                                                     self.coeffs[1])
-        if is1d:
-            return np.squeeze(xyw)
-        else:
-            return xyw
-        
+            self.coeffs.append(np.array(c,copy=False))
+
+    def __call__(self, x, y, c=None):
+        x -= self.shift[0]
+        y -= self.shift[1]
+        x *= self.scale[0]
+        y *= self.scale[1]
+        try:
+            import galsim
+            xw = galsim.utilities.horner2d(x, y, self.coeffs[0])
```

Comment:
It requires branch #865, currently in [code review](https://github.com/GalSim-developers/GalSim/pull/872).  That branch also includes the hooks to allow GalSim's WCS classes to be color-dependent, which I needed here.

I originally did copy over the horner2d code, but since I also wanted to use the galsim gnomonic projection (which I thought was probably too large to copy over), I switched it to the try/except structure for both.

But if you'd like to copy both of those bits over to this repo, I don't have any problem with that.